src/app/components/DeveloperCard/DeveloperCard.tsx

```diff
@@ -0,0 +1,29 @@
+import styles from "./DeveloperCard.module.css";
+
+interface DeveloperCardProps {
+  title: string;
+  description: string;
+  topics: string[];
+}
+
+export function DeveloperCard({
+  title,
+  description,
+  topics,
+}: DeveloperCardProps) {
+  return (
+    <div className={styles.card}>
```

Comment:
```suggestion
  const router = useRouter();

  const handleCardClick = (title: string) => {
    router.push(`/selection/topics?topic=${title}`);
  };

  return (
    <div className={styles.card} onClick={() => handleCardClick(title)}>
```

저는 이렇게 핸들러를 넣는게 컴포넌트 역할이 더 명확하게 보인다고 생각해요